### PR TITLE
add site specific settings for each datadog site

### PR DESCRIPTION
**Pull request by irabinovitch**
### What does this PR do?

Adding docs for how to point dogshell at non US1 Datadog sites.

Note that the --api_host command line argument doesn't not yet seem to support API1 and DD-GOV, but dogrc does.  (see https://github.com/DataDog/datadogpy/issues/771)

### Motivation

Someone asked me how to do it.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md [hunk 1]
```diff
@@ -48,11 +48,50 @@ Next, use your `dog` commands to quickly use the Datadog API from your terminal/
...
+api_host = https://us3.datadoghq.com
+```
+{{< /site-region >}}
+{{< site-region region="us5" >}}
+```text
+apikey = <DATADOG_API_KEY>
+appkey = <YOUR_APPLICATION_KEY>
+api_host = https://us5.datadoghq.com
+{{< /site-region >}}
+```
```
Comment (rtrieu) [inline content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md:72]:
```suggestion
```

File: content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md [hunk 2]
```diff
@@ -48,11 +48,50 @@ Next, use your `dog` commands to quickly use the Datadog API from your terminal/
...
+apikey = <DATADOG_API_KEY>
+appkey = <YOUR_APPLICATION_KEY>
+api_host = https://us3.datadoghq.com
+```
+{{< /site-region >}}
+{{< site-region region="us5" >}}
+```text
+apikey = <DATADOG_API_KEY>
+appkey = <YOUR_APPLICATION_KEY>
+api_host = https://us5.datadoghq.com
```
Comment (rtrieu) [inline content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md:70]:
```suggestion
api_host = https://us5.datadoghq.com
```
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
This looks good to me! The only thing was one closing tag for the text was outside a region tag.

File: content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md (hunk 1 above)
Comment (rtrieu) [reply content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md:72]: To clarify, my feedback here is that the closing tag for the text is out of place - it should be above the /site-region tag 😄

**Review by hestonhoffman — APPROVED** [review]
LGTM!

**Pull request merged**